lib/polyfill/pip_webkit.js

```diff
@@ -47,6 +47,16 @@ shaka.polyfill.PiPWebkit = class {
       return;
     }
 
+    const video =
+      /** @type {!HTMLVideoElement} */ (document.createElement('video'));
+    const isPipPresentationModeSupported =
+      video.webkitSupportsPresentationMode('picture-in-picture');
+
+    if (!isPipPresentationModeSupported) {
```

Comment:
There is a similar check below in requestPictureInPicture_.  Is it still needed?  Is it possible for webkitSupportsPresentationMode('picture-in-picture') to return true for one element and false for another?

Reply:
The main issue with this is the initial patching of `document.pictureInPictureEnabled ` so the check needs to be done before patching it.
As iOS doesn’t have video level disabling of PiP then any video will return false anyway.
As far as I tested on macOS and iPadOS all video elements will return true and there’s no api to disable it.

Reply:
If there's no way to disable it, you could remove the other check since you already checked here.